ui/src/dashboards/components/DashboardHeader.js

```diff
@@ -59,6 +62,16 @@ const DashboardHeader = ({
                   Rename
                 </button>
               : null}
+            {dashboard
```

Comment:
good call @lukevmorris. could you check https://github.com/influxdata/chronograf/issues/1372 and add/amend where necessary? that's the running list of these kinds of refactoring.